phenol-analysis/src/test/java/org/monarchinitiative/phenol/analysis/stats/mtc/BenjaminiYekutieliTest.java

```diff
@@ -19,7 +19,7 @@
  *  [1] 0.004977343 0.009954687 0.031523175 0.118211908 0.200089208 0.211892623 0.211892623 0.214025770 0.253844518 1.000000000 1.000000000
  * [12] 1.000000000 1.000000000 1.000000000 1.000000000
  */
-class BenjaminiYekutieliTest {
+public class BenjaminiYekutieliTest {
```

Comment:
Do these really still have to be public to run using the module system?
...
...
a few hours later...
Well, it turns out that IntelliJ 2021.3 no longer complains about a duplicate `module-info.java` in the `src/test/java` directory so problem solved right? See https://youtrack.jetbrains.com/issue/IDEA-253066

... but now a heap of other things are FUBAR

So it still seems like modules and tests just don't belong together in this world. Back to the freelove classpath party!

Reply:
No real experience with this yet, but this blog seems useful
https://sormuras.github.io/blog/2018-09-11-testing-in-the-modular-world.html
If I understand correctly, we might need to have module-info files inthe testing directories as well as "open module" statements

Reply:
But no arguments about ``this will need a beer``

Reply:
I tried the `open module ...` in a `module-info.java` in `src/test/java` in svart which tests perfectly on the classpath, but there were so many IntelliJ issues it was unworkable, which is what drove me to beer.

That blog did indeed seem useful and in a pure maven and junit world it seems possible, but for whatever reason IntelliJ has seriously ballsed-up with this. Given the utility of IntelliJ, even if the lower-level tooling works, this still isn't quite ready.

Reply:
I'm not 100% convinced that making test classes and methods public is a wrong thing to do. For me it is the lowest price to pay in order to make the tests work in the module system. Something must be done, otherwise the tests crash (easy to see if a class/method is not `public`). I do not see any reason why this should not be done. 
It's not super nice, but test classes are in `<scope>test</scope>`, and should not be available in the production code, right? 
Unless somebody forgets to use `<includeScope>compile</includeScope>` when using `maven-dependency-plugin`.

Reply:
One might also argue, what's the problem with simply using the classpath rather than module path in test code. Several - many I'd argue, but this isn't going to be solved here.